### PR TITLE
[modular][ready]imperial uniforms for command

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

image
image
image

## Why It's Good For The Game

flavour good, and gandalf wanted em
from star-wars-13-yog

## Changelog
:cl:
expansion: imperial clothing for command staff, a goggle helmet, and some more boots
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
I don't want more meta-humour. And these clothes don't really fit regardless.

**Comment by thestubborn** [comment]
> 
> 
> I don't want more meta-humour. And these clothes don't really fit regardless.

gandalf said ages back to add imperial clothing
I said I would

**Comment by ghost** [comment]
Ngl the CE one specifically is too on-the-nose. Having that one for the Imperial Enclave or just events might be fitting, but having CEs with it might just cause le funny 'deathstar cannon BSA operator'
The red one is also just too brightly colored aaa

Also wow. Hi it's orion, I'm on the wrong account

**Review by ghost — CHANGES_REQUESTED** [review]
Navel is the bellybutton, Naval is the military branch. Fix them typos :b

**Review by Gandalf2k15 — APPROVED** [review]

File: modular_skyrat/modules/customization/modules/client/loadout/head.dm [hunk 1]
```diff
@@ -202,6 +202,22 @@
 /datum/loadout_item/head/job
 	subcategory = LOADOUT_SUBCATEGORY_JOB
 
+/datum/loadout_item/head/job/captain/imperial
+	name = "Captain's Naval Cap."
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/head.dm]:
```suggestion
	name = "Captain's Naval Cap"
```

File: modular_skyrat/modules/customization/modules/client/loadout/head.dm [hunk 2]
```diff
@@ -202,6 +202,22 @@
...
+	path = /obj/item/clothing/head/imperial/cap
+	restricted_roles = list("Captain")
+
+/datum/loadout_item/head/job/hop/imperial
+	name = "Head of Personnel's Naval Cap."
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/head.dm]:
```suggestion
	name = "Head of Personnel's Naval Cap"
```

File: modular_skyrat/modules/customization/modules/client/loadout/head.dm [hunk 3]
```diff
@@ -220,11 +236,22 @@
 	restricted_desc = "Security"
 	restricted_roles = list("Warden","Detective","Security Medic","Security Sergeant","Security Officer","Head of Security","Civil Disputes Officer")
 
-/datum/loadout_item/head/job/navybluehosberet
+/datum/loadout_item/head/job/blasthelmet
+	name = "General Helmet"
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/head.dm]:
```suggestion
	name = "General's Helmet"
```

File: modular_skyrat/modules/customization/modules/client/loadout/head.dm [hunk 4]
```diff
@@ -220,11 +236,22 @@
...
+	restricted_desc = "Security and Command"
+	restricted_roles = list("Warden","Detective","Security Medic","Security Sergeant","Security Officer","Civil Disputes Officer","Captain", "Head of Personnel", "Blueshield", "Head of Security", "Research Director", "Quartermaster", "Chief Medical Officer", "Chief Engineer")
+
+/datum/loadout_item/head/job/hos/imperial
 	name = "Head of security's navyblue beret"
 	path = /obj/item/clothing/head/beret/sec/navyhos
 	restricted_roles = list("Head of Security")
 
+/datum/loadout_item/head/job/navybluehosberet
+	name = "Head of security's Naval Cap"
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/head.dm]:
```suggestion
	name = "Head of Security's Naval Cap"
```

File: modular_skyrat/modules/customization/modules/client/loadout/uniform.dm [hunk 5]
```diff
@@ -625,6 +635,11 @@
  	path = /obj/item/clothing/under/rank/security/head_of_security/formal
  	restricted_roles = list("Head of Security")
 
+/datum/loadout_item/uniform/job/imphos
+ 	name = "Head of Security's naval uniform"
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/uniform.dm]:
```suggestion
 	name = "Head of Security's Naval Uniform"
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 6]
```diff
@@ -299,3 +299,97 @@
...
+/obj/item/clothing/head/imperial/red
+	name = "red naval officer cap"
+	desc = "A red cap with a silver disk in the center."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "impred_cap"
+
+
+
+
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm]:
```suggestion

```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
https://gyazo.com/f09529eb26faceaeabfce7eae14f22c8
The Imperial Engineer might be better with black gloves/boots/plating rather than a full black uniform

**Comment by thestubborn** [comment]
> 
> 
> https://gyazo.com/f09529eb26faceaeabfce7eae14f22c8
> The Imperial Engineer might be better with black gloves/boots/plating rather than a full black uniform

i like that!

**Comment by ghost** [comment]
also unironically make the wedge helmet function as a crowbar 🤣

**Comment by ghost** [comment]
:cry: gandalf we werent done it needed the re-spriteeee

**Comment by Gandalf2k15** [comment]
Do it after :)

**Comment by ghost** [comment]
that thirsty for your imperial uniforms, i see 😏

**Pull request merged**